woocommerce/woocommerce-android
# [Better Tablet Products] Fixes for 2<->1 panes transitions

Closes: #10978


### Description

* Back arrow shown after 2 -> 1 pane transition on the product details screen
* If the product details screen was opened in 1 pane and then configuration changed to show 2 panes, then currently open product will be selected
* If after being in 2 pane mode, 1 pane mode shows products details, then back navigation will hide product details fragment and show product lists
(Infinite loading after coming back from details screen and config change is fixed in https://github.com/woocommerce/woocommerce-android/pull/11144)

### Testing instructions

Using a big phone, where in portrait mode 1 pane mode is used, and in landscape, 2 pane mode is used, try different combinations 1<->2 panes transactions and notice that they all make sense now

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/4923871/f2d890d7-2de2-486c-9f8d-cbbf0310c04b
